alembic/versions/063eeaf98c57_add_dealer_categories.py

```diff
@@ -0,0 +1,44 @@
+"""Add dealer categories
+
+Revision ID: 063eeaf98c57
+Revises: 167243c0e86c
+Create Date: 2017-07-21 03:24:02.771982
+
+"""
+
+
+# revision identifiers, used by Alembic.
+revision = '063eeaf98c57'
+down_revision = '167243c0e86c'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+
+try:
+    is_sqlite = op.get_context().dialect.name == 'sqlite'
+except:
+    is_sqlite = False
+
+if is_sqlite:
+    op.get_context().connection.execute('PRAGMA foreign_keys=ON;')
+    utcnow_server_default = "(datetime('now', 'utc'))"
+else:
+    utcnow_server_default = "timezone('utc', current_timestamp)"
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
You know, I don't even really care about these comments. But ever since Eli's [initial migration code review](https://github.com/magfest/ubersystem/pull/2447#discussion_r113351941), I've been nitpicking everyone to remove them.